### PR TITLE
Improve analytics docs

### PR DESCRIPTION
**Pull request by tsusanka**
As discussed this improves the analytics docs a bit to be more user-facing for those who wish to be more informative on the topic. We will link from Suite here. @matejkriz what needs to be finished is the Sentry part, let's discuss it on some occasion.

cc @tomasklim though not much has changed (just some moving around)

### CODE REVIEW

File: docs/analytics/sentry.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+Trezor Suite uses [Sentry.io](https://sentry.io/) to track errors and monitor app performance on the machines of users who have enabled anononymous data collection. This allows us to optimize Trezor Suite and fix compatibility issues across many different usage environments.
+
+**No data is shared with Sentry if users have disabled usage data tracking.**
+
+## What is being tracked
+
+### General data
+
+Browser (User Agent), System, Suite version, session id shared with analytics.
```
Comment (tomasklim) [inline docs/analytics/sentry.md]:
```suggestion
Browser (User Agent), System, Suite version, instance id shared with analytics.
```

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: docs/analytics/sentry.md (hunk 1 above)
Comment (tsusanka) [reply docs/analytics/sentry.md]: https://github.com/trezor/trezor-suite/pull/4516/commits/b572bcc799e9a9c49ea1e75c8af3b8f87e186d92

**Comment by tsusanka** [comment]
Feel free to push any improvements directly into this PR :).

**Comment by tsusanka** [comment]
I would like to revive this PR and merge it. @tomasklim please could you re-review to see if this looks alright to you?
- Please look carefully if I haven't removed anything from your recent changes. Rebasing is getting difficult as the changelog is in different place in `master`.
- In https://github.com/trezor/trezor-suite/commit/92d09a999ac151ea06b17358157031969cba73bd you have [removed](https://github.com/trezor/trezor-suite/commit/92d09a999ac151ea06b17358157031969cba73bd#diff-7b3f909f3381215b37e57d29571681527b287f3fb65a51f332a1af689665b422L276-L279) also the "Fixed" part of 1.9. That seems like a mistake isn't it? I have added it back, let me know what is right.

File: docs/analytics/aws-technical.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+-   Desktop build: https://data.trezor.io/suite/log/desktop/stable.log
+-   Web build: https://data.trezor.io/suite/log/web/stable.log
+
+Data from **development** builds are sent to:
+
+-   Desktop build: https://data.trezor.io/suite/log/desktop/develop.log
+-   Web build: https://data.trezor.io/suite/log/web/develop.log
+
+Data from localhost are not currently tracked anywhere.
+
```
Comment (tomasklim) [inline docs/analytics/aws-technical.md]:
```suggestion
```

File: docs/analytics/aws.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# AWS Analytics: Info & Changelog
+
+_For a deeper technical writeup of analytics processes intended for developers, please see [aws-technical](aws-technical.md)._
+
+Trezor Suite can be set to collect real-world data to improve the performance of both web and desktop apps. This anonymous data is only shared by users who have usage data tracking enabled.
+
+Data tracking can be toggled on or off at any time through the Trezor Suite settings menu.
+
+By default, analytics will be enabled immediately following the onboarding process, and the application will start to track events and errors. Users are given the option to disable data collection before completing the onboarding.
```
Comment (tomasklim) [inline docs/analytics/aws.md]: It is enabled immediately after you confirm
Screenshot 2022-03-15 at 17 03 08

**Review by tomasklim — CHANGES_REQUESTED** [review]

**Comment by tomasklim** [comment]
> I would like to revive this PR and merge it. @tomasklim please could you re-review to see if this looks alright to you?
> 
> * Please look carefully if I haven't removed anything from your recent changes. Rebasing is getting difficult as the changelog is in different place in `master`.
> * In [92d09a9](https://github.com/trezor/trezor-suite/commit/92d09a999ac151ea06b17358157031969cba73bd) you have [removed](https://github.com/trezor/trezor-suite/commit/92d09a999ac151ea06b17358157031969cba73bd#diff-7b3f909f3381215b37e57d29571681527b287f3fb65a51f332a1af689665b422L276-L279) also the "Fixed" part of 1.9. That seems like a mistake isn't it? I have added it back, let me know what is right.

a) We do track events from localhost to `develop.log` now
b) Yes, you are correct. I accidentally deleted more than I wanted. Thank you

File: docs/analytics/aws.md (hunk 3 above)
Comment (tsusanka) [reply docs/analytics/aws.md]: 954586dd6a

**Comment by tsusanka** [comment]
Done, please check it out!

**Review by tomasklim — APPROVED** [review]
Please squash it and let's merge it

**Pull request merged**